src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -1,3 +1,186 @@
+const { t } = useI18n();
+const { trackGoal, Goals } = useFathom();
+const { darkMode } = useDarkMode();
+const { upToLargeBreakpoint } = useBreakpoints();
+
+/**
+ * DATA
+ */
+const columns = ref<ColumnDefinition<DecoratedPoolWithShares>[]>([
+  {
+    name: 'Icons',
+    id: 'icons',
+    accessor: 'uri',
+    Header: 'iconColumnHeader',
+    Cell: 'iconColumnCell',
+    width: 125,
+    noGrow: true
+  },
+  {
+    name: t('composition'),
+    id: 'poolName',
+    accessor: 'id',
+    Cell: 'poolNameCell',
+    width: 350
+  },
+  {
+    name: t('myBalance'),
+    accessor: pool => fNum(pool.shares, 'usd', { forcePreset: true }),
+    align: 'right',
+    id: 'myBalance',
+    hidden: !props.showPoolShares,
+    sortKey: pool => Number(pool.shares),
+    width: 150,
+    cellClassName: 'font-numeric'
+  },
+  {
+    name: t('poolValue'),
+    accessor: pool => fNum(pool.totalLiquidity, 'usd'),
+    align: 'right',
+    id: 'poolValue',
+    sortKey: pool => {
+      const apr = Number(pool.totalLiquidity);
+      if (apr === Infinity || isNaN(apr)) return 0;
+      return apr;
+    },
+    width: 150,
+    cellClassName: 'font-numeric'
+  },
+  {
+    name: t('volume24h', [t('hourAbbrev')]),
+    accessor: pool => fNum(pool.dynamic.volume, 'usd'),
+    align: 'right',
+    id: 'poolVolume',
+    sortKey: pool => {
+      const apr = Number(pool.dynamic.volume);
+      if (apr === Infinity || isNaN(apr)) return 0;
+      return apr;
+    },
+    width: 175,
+    cellClassName: 'font-numeric'
+  },
+  {
+    name: t('apr'),
+    Cell: 'aprCell',
+    accessor: pool => pool.dynamic.apr.total,
+    align: 'right',
+    id: 'poolApr',
+    sortKey: pool => {
+      const apr = Number(pool.dynamic.apr.total);
+      if (apr === Infinity || isNaN(apr)) return 0;
+      return apr;
+    },
+    width: 150
+  },
+  {
+    name: t('migrate'),
+    Cell: 'migrateCell',
+    accessor: 'migrate',
+    align: 'right',
+    id: 'migrate',
+    width: 150,
+    hidden: !props.showMigrationColumn
+  }
+]);
+
+/**
+ * METHODS
+ */
+function orderedTokenAddressesFor(pool: DecoratedPoolWithShares) {
+  const sortedTokens = orderedPoolTokens(pool);
+  return sortedTokens.map(token => getAddress(token.address));
+}
+
+function orderedPoolTokens(pool: DecoratedPoolWithShares): PoolToken[] {
+  if (isStablePhantom(pool.poolType))
+    return pool.tokens.filter(token => token.address !== pool.address);
+  if (isStableLike(pool.poolType)) return pool.tokens;
+
+  const sortedTokens = pool.tokens.slice();
+  sortedTokens.sort((a, b) => parseFloat(b.weight) - parseFloat(a.weight));
+  return sortedTokens;
+}
+
+function handleRowClick(pool: DecoratedPoolWithShares) {
+  trackGoal(Goals.ClickPoolsTableRow);
+  router.push({ name: 'pool', params: { id: pool.id } });
+}
+
+function navigateToPoolMigration(pool: DecoratedPoolWithShares) {
+  router.push({
+    name: 'migrate-pool',
+    params: {
+      from: pool.id,
+      to: POOLS.IdsMap[networkId.value]?.bbAaveUSD
```

Comment:
hardcoded id, chould use a map as mentioned above

Reply:
@evgenyboxer did you look into this?